metasynth/var.py

```diff
@@ -146,19 +152,29 @@ def to_dict(self) -> Dict[str, Any]:
         return var_dict
 
     def __str__(self) -> str:
-        """Create a readable string from a variable."""
-        return str({
-            "name": self.name,
-            "description": self.description,
-            "type": self.var_type,
-            "dtype": self.dtype,
-            "prop_missing": self.prop_missing,
-            "distribution": str(self.distribution),
-        })
+        """Return an easy to read formatted string for the variable."""
+        description = f'Description: "{self.description}"\n' if self.description else ""
+
+        if self.distribution is None:
+            distribution_formatted = "No distribution information available"
+        else:
+            distribution_formatted = "\n".join(
+                "\t" + line for line in str(self.distribution).split("\n")
+            )
+
+        return (
+            f'"{self.name}"\n'
+            f'{description}'
+            f'- Variable Type: {self.var_type}\n'
+            f'- Data Type: {self.dtype}\n'
+            f'- Prop. Missing: {self.prop_missing}\n'
```

Comment:
I don't know if this is clear to all users, perhaps it's better to write it all out: "Proportion Missing Values"